### PR TITLE
Lesson4

### PR DESCRIPTION
**Pull request by iliijaro**

### CODE REVIEW

File: Lesson_1/HelloName_Lesson/HelloName_Lesson/Lesson4.cs [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+{
+	internal class Lesson4
+	{
+        /// <summary>
+        /// Заменить в строке все вхождения 'test' на 'testing'. Удалить из текста все символы, являющиеся цифрами.
+        /// </summary>
+        
+        public static void HomeTask4()
+		{
+            string NewString = "test and 1test 231231 best test";
```
Comment (VladimirVolkovVD) [inline Lesson_1/HelloName_Lesson/HelloName_Lesson/Lesson4.cs]: name of the local variable starting from lowercase should be newString
change please for all variables in the code

**Review by VladimirVolkovD — COMMENTED** [review]
change variables name
tasks  is good

File: Lesson_1/HelloName_Lesson/HelloName_Lesson/Lesson4.cs (hunk 1 above)
Comment (iliijaro) [reply Lesson_1/HelloName_Lesson/HelloName_Lesson/Lesson4.cs]: According to the CodeStyle, we can use both variants, isn't it?
CodeStyle

Comment (iliijaro) [reply Lesson_1/HelloName_Lesson/HelloName_Lesson/Lesson4.cs]: but anyway, I've changed everything to the unified style

**Pull request merged**